src/theory/quantifiers/expr_miner.h

```diff
@@ -92,6 +92,12 @@ class ExprMiner
                          ExprManagerMapCollection& varMap,
                          Node query,
                          bool& needExport);
+  /**
+   * Run the satisfiability check on query, without returning the checker. This
+   * can be used for cases where for instance the model for the query is not
+   * important.
+   */
```

Comment:
I think you mean this comment in opposition to the above method but it's very cryptic. I think it would be clearer to just say that you just do the check and return the result.